.github/workflows/test.yml

```
@@ -69,40 +69,36 @@ jobs:
+      - name: Set up Rust
+        uses: dtolnay/rust-toolchain@stable
         with:
-          python-version: '3.10'
+          components: rustfmt, clippy
 
-      - name: Cache Python dependencies
+      - name: Install executor Rust system dependencies
+        run: sudo apt-get update && sudo apt-get install -y pkg-config libssl-dev
+
+      - name: Cache Rust dependencies
         uses: actions/cache@v3
```

actionlint (1.7.12)

[error] the runner of "actions/cache@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)